CHANGELOG.md

```diff
@@ -3,6 +3,22 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [3.3.0] - 2022-02-23
+
+This minor release fixes a bug in the maxHeaderSize parameter that could lead to panics.  We introduce an external magic number to keep track of external dependencies.  We bump up the minimum required Go version to 1.19.  No changes were made to the format of data on disk.
```

Comment:
limit line width

Reply:
fix spaces before `We introduce`

Reply:
is line width in scope for the changelog?  The summary will not appear very good with short lines, and our previous releases have lines that wrap around.

Reply:
I think markdown cleans up newlines while rendering. It shouldn't be a problem

Reply:
I tried - it doesn't, it will render on multiple lines